webapp-django/crashstats/crashstats/static/crashstats/css/daily.less

```diff
@@ -117,6 +117,63 @@ table.graph {
 #sig-history-legend {
     float: left;
 }
+.crashes-per-adi-legend {
+    width: 100%;
+    text-align: center;
+}
+/* Colours to override metricsgraphics defaults */
+.mg-line1-color {
+    stroke: #6a3d9a;
+}
+.mg-hover-line1-color {
+    stroke: #6a3d9a;
+}
+.mg-line2-color {
+    stroke: #e31a1c;
+}
+.mg-hover-line2-color {
+    stroke: #e31a1c;
+}
+.mg-line3-color {
+    stroke: #008800;
+}
+.mg-hover-line3-color {
+    stroke: #008800;
+}
+.mg-line4-color {
+    stroke: #1f78b4;
+}
+.mg-hover-line4-color {
+    stroke: #1f78b4;
+}
+.mg-line1-legend-color {
+    color: #6a3d9a;
+    fill: #6a3d9a;
+}
+.mg-line2-legend-color {
+    color: #e31a1c;
+    fill: #e31a1c;
+}
+.mg-line3-legend-color {
+    color: #008800;
+    fill: #008800;
+}
+.mg-line4-legend-color {
+    color: #1f78b4;
+    fill: #1f78b4;
+}
+.mg-area1-color {
+    fill: #6a3d9a;
+}
+.mg-area2-color {
+    fill: #e31a1c;
+}
+.mg-area3-color {
+    fill: #008800;
+}
+.mg-area4-color {
+    fill: #1f78b4;
+}
```

Comment:
It seems not... These colours should be added to the new css file for customising metricsgraphics classes (so e.g. the home page graph will get the same colours). That means this is now dependent on https://github.com/mozilla/socorro/pull/2893